sunpy/coordinates/tests/test_frames.py

```diff
@@ -13,10 +13,15 @@
 )
 from astropy.tests.helper import assert_quantity_allclose
 
+from sunpy import sun
+from sunpy.coordinates.frames import (
+    Heliocentric,
+    HeliographicCarrington,
+    HeliographicStonyhurst,
+    Helioprojective,
+)
 from sunpy.time import parse_time
 from sunpy.util.exceptions import SunpyUserWarning
-from ... import sun
-from ..frames import Heliocentric, HeliographicCarrington, HeliographicStonyhurst, Helioprojective
```

Comment:
Can you pull these import changes and the test changes out of this PR and into another one?

Reply:
Sure.